pkg/cmdimpl/add.go

```diff
@@ -433,15 +471,27 @@ func Add(args []string, allParams AddParamSet, deps *AddDependencies) error {
 		}
 
 		// Create flux custom resources for new repo being added
-		source, err := generateSourceManifest()
-		if err != nil {
-			return wrapError(err, "could not generate source manifest")
+		var source []byte
+		if params.HelmRepository != "" {
+			source, err = generateSourceManifestHelm()
+			if err != nil {
+				return wrapError(err, "could not generate source manifest")
+			}
+		} else {
+			source, err = generateSourceManifestGit()
```

Comment:
huge favor, could you please help me make this function to use the new Chart field instead of Path. When I created it, I used `Path` but it seems more accurate to use `Chart` now.

Reply:
I talked with @jrryjcksn  at least on whether or not i should use path. We agreed for helm-repository chart made sense but I didnt really ask about using chart for github.

Reply:
Looks like Flux asks the user for "chart" in either case so we should probably follow that lead.